resource-watch/resource-watch
# Explore detail: add support for datasets without layers

## Overview
This PR adds support for the visualization on Explore detail of datasets that have no layers associated to them.

## Testing instructions
Check the explore detail page with a dataset that has no layers, such as e.g. http://localhost:9000/data/explore/foo046-Food-Footprint-in-Calories-WRR-update
